build/OutputDirectories.props

```diff
@@ -25,5 +25,15 @@
       <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
       <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
       <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <ArchiveOutputDirectory>$(PackagesDirectory)</ArchiveOutputDirectory>
```

Comment:
Are you ok with letting it go for this PR (since it's a pretty big one). Once it goes through I'll take a pass at cleaning up our global props files. Basically if we have duplicate properties I'll remove them, and if a property is used in only one file I'll move it back into the file.